data.md

```diff
@@ -0,0 +1,106 @@
+
+## Our Websites and Hosted Services
+
+### What data we collect
+
+**System logs** may contain:
+
+- Timestamp of visit
+- IP address
+- Operating system
+- Browser type
+- User-specified URL to access the service
+
+**User accounts** may contain:
+
+- First and last name
+- Email address
+
+**Videoconferencing data** may contain:
+
+- Network usage information and aggregated statistics, such as number of users, number of meetings, number of video and audio streams, session lengths, etc.
+- Chat logs, notes, and uploaded content during meetings, such as presentations and images
+- Phone numbers that connect to meetings (if audio connection is made via a telephone call)
+
+We do not knowingly collect any information from children under the age of 13 without appropriate consent and authorization.
+If you believe we have inadvertently collected personal information from a child under 13 without proper consents please contact `data [at] hypha.coop` so that we may delete such information as soon as possible.
+
+### Why we collect data
+
+We keep standard system logs for **monitoring of system performance and debugging purposes**.
+
+Some of our hosted services automatically log the time you access your account on our systems, and may store "cookies" on your computer.
+"Cookies" are small pieces of information that a website sends to your computer's hard drive while you are viewing a website, which may stay on your computer until you delete them.
+These are used by some of our hosted services to **deliver a more personal and interactive experience**.
+
+In general, personal information you submit to us is used either to **respond to requests that you make, or to aid us in serving you better**.
+We do our best to protect the personal information you share with us, and do not share them with third parties.
+We may however, **display and share anonymized feedback or personal testimonials on our website and social media, or in other publicly viewable places**.
+
+### Where we keep data
+
+Our services are hosted on servers managed by third-party service providers, physically located in Canada.
+These servers are accessible to members of Hypha, and specific collaborators as deemed necessary to deliver the services and content.
+Where necessary, servers may also be accessed by other parties to comply with laws or to respond to lawful requests and legal process and in an emergency to protect the personal safety of any person.
+
+### How long we keep data
+
+System logs are generally retained for a duration that is appropriate for the particular system.
+
+User accounts necessary to deliver services and content are generally kept indefinitely, and periodically backed up, until the account holders have requested to delete them.
+Deleting your account on a particular service generally does not mean all your information is deleted from our periodic backups.
+If you have a concern, please email `data [at] hypha.coop`.
```

Comment:
look at how we stylize elsewhere, I'm also not particularly worried about emails in plaintext (e.g., data@hypha.coop)